admin_guide/cluster_management.rst

```diff
@@ -127,3 +127,45 @@ place for the master node, users can dynamically reconstruct this metadata from
 shard information available on the worker nodes. To learn more about the metadata
 tables and their schema, you can visit the :ref:`metadata_tables` section of our documentation.
 
+Tenant Isolation
+################
+
+Citus places table rows into worker shards based on the hashed value of the rows' distribution column.  Multiple distribution column values often fall into the same shard. In the Citus multi-tenant use case this means that tenants often share shards.
+
+Many multi-tenant SaaS providers face the problem that some tenants grow much bigger than the others, or require special resource guarantees. In Citus Enterprise Edition large tenants can be *isolated* to their own dedicated shards to better control the resource allocation between small and large tenants.
+
+Every shard is marked in Citus metadata with the range of hashed values it contains (more info in the reference for :ref:`pg_dist_shard <pg_dist_shard>`). The Citus UDF :code:`isolate_tenant_to_new_shard` removes rows with a given distribution value from their current shard placement and creates a shard which will hold them separately from other rows. Furthermore, it takes a :code:`CASCADE` option which isolates not just the table but all tables :ref:`co-located <colocation>` with it.
+
+.. code-block:: postgresql
+
+  -- This query creates an isolated shard for the given tenant_id and
+  -- returns the new shard id.
+
+  SELECT isolate_tenant_to_new_shard('table_name', tenant_id);
+
+  -- For example
+
+  SELECT isolate_tenant_to_new_shard('lineitem', 135);
+
+  -- If the given table has co-located tables, the query above errors out and
+  -- advises to use the CASCADE option
+
+  SELECT isolate_tenant_to_new_shard('lineitem', 135, 'CASCADE');
+
+Once a tenant is isolated to a dedicated shard it can be moved to a separate node in the Citus cluster for true hardware isolation. As mentioned, the :code:`isolate_tenant_to_new_shard` function returns the newly created shard id, and this id can be used to move the shard:
```

Comment:
Maybe clarify that the function will create a shard, but places it on the same node as the original shard? That can be a preamble for the rest of this. We don't note that point anywhere.

Could we also note that this move function (along with tenant-isolation) both belong in Citus Enterprise only?